activerecord/activerecord.gemspec

```diff
@@ -37,4 +37,5 @@ Gem::Specification.new do |s|
 
   s.add_dependency "activesupport", version
   s.add_dependency "activemodel",   version
+  s.add_dependency "timeout", ">= 0.4.0"
```

Comment:
I'd failed to appreciate that we'd be doing this... it's fine--and necessary--but it does make me again wish we could say "we don't need this gem, but _if it's present_, we need this condition satisfied"

Reply:
>  but if it's present, we need this condition satisfied"

Well, it's part of the default gems so always present.